README.md

```diff
@@ -9,106 +9,236 @@
 - Document how a component behaves in various scenarios.
 - Get a11y, snapshot and visual tests using other addons for free.
 
+[Full documentation and live demos](https://msw-sb.vercel.app/)
+
+## Installing and Setup
+
+### Install MSW and the addon
+
+```sh
+npm i -D msw msw-storybook-addon
+```
+
+### Generate service worker for MSW in your public folder.
+
+_If you already use MSW in your project, you have likely done this before so you can skip this step._
+
+```sh
+npx msw init public/
+```
+
+Refer to the [MSW official guide](https://mswjs.io/docs/getting-started/integrate/browser) for framework specific paths if you don't use `public`.
+
+### Enable MSW in Storybook by adding this to `./storybook/preview.js`
+
+```js
+import { initialize, mswDecorator } from 'msw-storybook-addon';
+
+// Initialize MSW
+initialize();
+
+// Provide the MSW addon decorator globally
+export const decorators = [mswDecorator];
+```
+
+### Start Storybook
+
+When running Storybook, you have to serve the `public` folder as an asset to Storybook. Refer to [the docs](https://storybook.js.org/docs/react/configure/images-and-assets) if needed.
+
+```sh
+npm run start-storybook -s public
+```
+
+## Usage
+
+You can pass request handlers (https://mswjs.io/docs/basics/request-handler) into the `handlers` property of the `msw` parameter. This is commonly an array of handlers.
+
 ```js
 import { rest } from 'msw'
 
 export const SuccessBehavior = () => <UserProfile />
 
 SuccessBehavior.parameters = {
   msw: {
-      handlers: [
-          rest.get('/user', (req, res, ctx) => {
-              return res(
-                  ctx.json({
-                      firstName: 'Neil',
-                      lastName: 'Maverick',
-                  })
-              )
+    handlers: [
+      rest.get('/user', (req, res, ctx) => {
+        return res(
+          ctx.json({
+            firstName: 'Neil',
+            lastName: 'Maverick',
           })
-      ],
-  }
+        )
+      }),
+    ]
+  },
+}
+```
+
+**Deprecated usage**
+
+From previous versions, the msw parameter takes an array of handlers. **This is not recommended** anymore and is deprecated. You should use the `handlers` instead, as stated above.
```

Comment:
Do you think documenting deprecations is worth it? 

I like to think of any documentation as the guidelines I can (and likely should) follow. Including deprecation mentions _and examples_ may do more harm than use. For example, somebody may be browsing through the docs in a hurry and their eyes would stop on the first seemingly plausible example. Too bad if it turns out to be a deprecated usage example.

Reply:
I agree. removing!